modules/core/client/views/header.client.view.html

```diff
@@ -36,17 +36,8 @@
           <span ng-bind="authentication.user.displayName"></span> <b class="caret"></b>
         </a>
         <ul class="dropdown-menu" role="menu">
-          <li ui-sref-active="active">
-            <a ui-sref="settings.profile">Edit Profile</a>
-          </li>
-          <li ui-sref-active="active">
-            <a ui-sref="settings.picture">Change Profile Picture</a>
-          </li>
-          <li ui-sref-active="active" ng-show="authentication.user.provider === 'local'">
-            <a ui-sref="settings.password">Change Password</a>
-          </li>
-          <li ui-sref-active="active">
-            <a ui-sref="settings.accounts">Manage Social Accounts</a>
+          <li ui-sref-active="active" ng-repeat="item in accountMenu.items">
```

Comment:
I just used the same that was there for the rest of the menus:

https://github.com/trainerbill/mean/blob/MenusFixes/modules/core/client/views/header.client.view.html#L13